force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,15 @@
+trigger AccountTrigger on Account (before insert, before update) {
+    if(Trigger.isBefore){
+        if(Trigger.isInsert){
+            List<Account> accounts = trigger.new;
+            for(Account anAccount : accounts){
+                AccountValidation.validateAccount(anAccount, false);
```

Comment:
validate account dentro tiene una query, esto seria una query dentro de un for, deberias bulkificar validateAccount